### DISCUSSION TITLE
Sometimes mobile can't clear author filter

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#44239] [Mobile] [answered]
GitHub Mobile beta
Open community/community 
Open discussions
Filter by author and category
Try to clear the author filter

https://user-images.githubusercontent.com/2119212/212366115-5eaf7e50-f428-477e-8e2e-3c0a2dc073d0.mp4

**Comment by stevepopovich** [comment, marked as answer]
Hi @jsoref thanks for filing! I passed this along to the team for fixing.
